src/System Application/App/AppSource Gallery/src/AppSourceProductManager.Codeunit.al

```diff
@@ -282,7 +371,7 @@ codeunit 2515 "AppSource Product Manager" implements "IAppSource Product Manager
         SetCommonHeaders(RestClient);
         RestClient.SetDefaultRequestHeader('client-request-id', ClientRequestID);
 
-        exit(Dependencies.RestClient_GetAsJSon(RestClient, RequestUri).AsObject());
+        exit(Dependencies.GetAsJSon(RestClient, RequestUri).AsObject());
```

Comment:
So now i see Dependencies. all over the code and i have no idea what it is... Please name your variables after the thing it points to...